netbox/overlays/ocp-prod/postgresclusters/netbox.yaml

```diff
@@ -41,3 +50,11 @@ spec:
             resources:
               requests:
                 storage: 15Gi
+      - name: repo2
+        schedules:
+          # full backups on the first of the month
+          full: '17 1 1 * *'
+          # differential backups once/week
+          differential: '17 2 * * 0'
+          # incremental backups daily
+          incremental: '17 4 * * *'
```

Comment:
@larsks sorry I am asking after I merged it. But shouldn't `repo2` specify `s3` as the backup storage system with `bucket`, `endpoint` and `region`?

Reply:
...oh, but we're probably missing the repository type.

Reply:
I was looking at this example: https://github.com/naved001/postgres-operator-examples/blob/main/kustomize/s3/postgres.yaml#L25

Any reason why we would want to put the endpoint, bucket, region in the secret instead of the way they did in the example?

Reply:
Mostly I was just trying to avoid splitting the configuration into too may difference places.